disnake/embeds.py

```diff
@@ -302,6 +316,27 @@ def __bool__(self) -> bool:
             )
         )
 
+    def __eq__(self, other: Any) -> bool:
+        return isinstance(other, Embed) and all(
+            (
+                (self.title or None) == (other.title or None),
```

Comment:
What's the purpose of `or None` here?

Reply:
Oh, I see the point now. Taking this as an example:
```py
e = disnake.Embed(title="h", url="")
m = await ctx.send(embed=e)
print(m.embeds[0] == e)
```
This'll print `False` (ignoring the `color` issue mentioned below), since `m.embeds[0].url == None` but `e.url == ""`.

Reply:
I vaguely know how to implement it but I don't think I can do it in one line as how it is done now